src/theory/arith/constraint.h

```diff
@@ -473,6 +476,10 @@ class Constraint {
     return d_literal;
   }
 
+  // Gets a literal in the normal form suitable for proofs.
+  // That is, (sum of non-const monomials) >< const.
+  Node getProofLiteral() const;
```

Comment:
This is basically converting the constraint to a Node?  Would a better name be `toProofLiteral()`?

Reply:
The name mirrors `getLiteral`, a method which returns the externally-valid literal associated with this constraint.

I like the connection between the names, but I'm open to changing it.

Reply:
Ah I see, that's fine then.